reference/forms/types/options/empty_data.rst.inc

```diff
@@ -10,11 +10,12 @@ empty_data
 DEFAULT_PLACEHOLDER
 
 This option determines what value the field will return when the submitted
-value is empty.
+value is empty. It does not set an initial value if none is provided when
+the form is rendered in a view (see `Data` or `Placeholder` options).
 
-But you can customize this to your needs. For example, if you want the
-``gender`` choice field to be explicitly set to ``null`` when no value is
-selected, you can do it like this::
+It helps you handling form submission and you can customize this to your needs.
```

Comment:
We could maybe remove "customize" as this is the point of an option.
It could be  "It helps you handling form submission, for example, [...]"